Portal/src/Datahub.Portal/Startup.cs

```diff
@@ -292,7 +302,8 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env, ILogger<
         
         app.Use(async (context, next) =>
         {
-            context.Response.Headers.Append("X-Frame-Options", "SAMEORIGIN");
+            context.Response.Headers.Append("X-Frame-Options", "DENY");
```

Comment:
```suggestion
            context.Response.Headers.Append("Content-Security-Policy", "frame-ancestors 'none'");
```

I like your original suggestion of the CSP

Reply:
I was not able to make the CSP work correctly. Doing a proper CSP would require some auditing and more experimentation